scripts/ec2cleaning.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from utils.providers import list_providers, get_mgmt
+from time import sleep
+date = datetime.now()
+deletetime = 60 * 60 * 24 * 7  # 7 Days
+
+
```

Comment:
@mmojzis we may need params added here. for reference, check "integration_tests/scripts/cleanup_old_vms.py"
all the delete methods needs to handle the param values, if specified else should have default values.
Once you add the params here, we can discuss deletion rules.